README_ID.md

```diff
@@ -0,0 +1,105 @@
+# [OFFICIAL] Member baru, Rekan kerja dan Tim -> Gabung disini !!! 🚀🚀🚀
+<!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
+[![All Contributors](https://img.shields.io/badge/all_contributors-6-orange.svg?style=flat-square)](#contributors-)
+<!-- ALL-CONTRIBUTORS-BADGE:END -->
+
+[![README.md](https://img.shields.io/badge/English-up-brightgreen)](README.md)
```

Comment:
You will need to add your language here as a button link.  See the next 2 lines as well below